### PR TITLE
Videos watchlater

### PR DESCRIPTION
**Pull request by Naveenchandar**
screencapture-localhost-3000-watchlater-2022-04-27-22_33_27

screencapture-localhost-3000-watchlater-2022-04-27-22_33_12

screencapture-localhost-3000-watchlater-2022-04-27-22_32_51

* Added watch later options to videos
* Watch later page added
* Added watch later videos with options

### CODE REVIEW

**Review by ranitasahaa1230 — APPROVED** [review]
Good breakdown of components
Clean and readable code
Good for approval

File: src/components/sidebar/index.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ export function Sidebar() {
                 <li><NavLink to='/' className={({ isActive }) => activeLinkClassName(isActive)}>Home</NavLink></li>
                 <li><NavLink to='/explore' className={({ isActive }) => activeLinkClassName(isActive)}>Explore</NavLink></li>
                 <li><NavLink to='/' className={({ isActive }) => activeLinkClassName(isActive)}>Playlists</NavLink></li>
-                <li><NavLink to='/' className={({ isActive }) => activeLinkClassName(isActive)}>Watch later</NavLink></li>
+                <li><NavLink to='/watchlater' className={({ isActive }) => activeLinkClassName(isActive)}>Watch later</NavLink></li>
                 <li><NavLink to='/' className={({ isActive }) => activeLinkClassName(isActive)}>Liked videos</NavLink></li>
                 <li><NavLink to='/' className={({ isActive }) => activeLinkClassName(isActive)}>History</NavLink></li>
             </ul>
```
Comment (rohanmathur91) [inline src/components/sidebar/index.js]: try to put paths in an array and then just loop over it

**Review by rohanmathur91 — APPROVED** [review]
- Liked the file structuring 
- Good use of custom hooks
- Over all looks good

File: src/components/sidebar/index.js (hunk 1 above)
Comment (Naveenchandar) [reply src/components/sidebar/index.js]: Sure.

**Pull request merged**